frontend/src/components/account-settings.ts

```diff
@@ -311,6 +311,9 @@ export class AccountSettings extends LiteElement {
             name="newPassword"
             type="password"
             label="${msg("New password")}"
+            help-text=${msg("Must be between 8-64 characters")}
+            minlength="8"
+            maxlength="64"
```

Comment:
Maybe we should leave out maxlength so that passwords aren't truncated without the users' knowledge if they're copy-pasting a password? (not sure if password managers check maxlength when generating a password, either.) @Shrinks99 to weigh in?

Reply:
> not sure if password managers check maxlength when generating a password, either.

If they're smart they should!  I always use the KeePassXC client and set the randomizer to generate them based on the requirements so I don't personally run into this problem. 🤷

In any case, passwords should not truncate without the user's knoweldge.  This should result in an error with a message stating the password is longer than 64 characters and must be shortened.

Reply:
Okay, if we want to return an error I think @SuaYoo is right that we remove `maxlength`

Reply:
Done!